esmvaltool/recipes/recipe_ocean_bgc.yml

```
@@ -113,6 +113,14 @@ preprocessors:
 
 
   prep_global_profile:
+    annual_mean:
+    average_region:
+      coord1: longitude
+      coord2: latitude
+
+  prep_global_profile_decadal:
+    annual_mean:
+       decadal: True

```

```suggestion
       decadal: true
```
please check your recipes using `yamllint`